{"numFailedTestSuites":1,"numFailedTests":6,"numPassedTestSuites":0,"numPassedTests":0,"numPendingTestSuites":0,"numPendingTests":0,"numRuntimeErrorTestSuites":0,"numTodoTests":0,"numTotalTestSuites":1,"numTotalTests":6,"openHandles":[],"snapshot":{"added":0,"didUpdate":false,"failure":false,"filesAdded":0,"filesRemoved":0,"filesRemovedList":[],"filesUnmatched":0,"filesUpdated":0,"matched":0,"total":0,"unchecked":0,"uncheckedKeysByFile":[],"unmatched":0,"updated":0},"startTime":1747963761919,"success":false,"testResults":[{"assertionResults":[{"ancestorTitles":["Auth Controller","POST /api/v1/auth/register"],"duration":77,"failureDetails":[{}],"failureMessages":["MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:220:13)\n    at NativeConnection.openUri (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\connection.js:766:34)\n    at Mongoose.connect (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\index.js:416:15)\n    at C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:21:20\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"],"fullName":"Auth Controller POST /api/v1/auth/register should register a new user","invocations":1,"location":null,"numPassingAsserts":0,"retryReasons":[],"status":"failed","title":"should register a new user"},{"ancestorTitles":["Auth Controller","POST /api/v1/auth/register"],"duration":4,"failureDetails":[{}],"failureMessages":["MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:220:13)\n    at NativeConnection.openUri (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\connection.js:766:34)\n    at Mongoose.connect (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\index.js:416:15)\n    at C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:21:20\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"],"fullName":"Auth Controller POST /api/v1/auth/register should not register a user with duplicate email","invocations":1,"location":null,"numPassingAsserts":0,"retryReasons":[],"status":"failed","title":"should not register a user with duplicate email"},{"ancestorTitles":["Auth Controller","POST /api/v1/auth/register"],"duration":2,"failureDetails":[{}],"failureMessages":["MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:220:13)\n    at NativeConnection.openUri (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\connection.js:766:34)\n    at Mongoose.connect (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\index.js:416:15)\n    at C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:21:20\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"],"fullName":"Auth Controller POST /api/v1/auth/register should not register a user with missing required fields","invocations":1,"location":null,"numPassingAsserts":0,"retryReasons":[],"status":"failed","title":"should not register a user with missing required fields"},{"ancestorTitles":["Auth Controller","POST /api/v1/auth/login"],"duration":3,"failureDetails":[{}],"failureMessages":["MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:220:13)\n    at NativeConnection.openUri (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\connection.js:766:34)\n    at Mongoose.connect (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\index.js:416:15)\n    at C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:21:20\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"],"fullName":"Auth Controller POST /api/v1/auth/login should login an existing user with correct credentials","invocations":1,"location":null,"numPassingAsserts":0,"retryReasons":[],"status":"failed","title":"should login an existing user with correct credentials"},{"ancestorTitles":["Auth Controller","POST /api/v1/auth/login"],"duration":2,"failureDetails":[{}],"failureMessages":["MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:220:13)\n    at NativeConnection.openUri (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\connection.js:766:34)\n    at Mongoose.connect (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\index.js:416:15)\n    at C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:21:20\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"],"fullName":"Auth Controller POST /api/v1/auth/login should not login with incorrect password","invocations":1,"location":null,"numPassingAsserts":0,"retryReasons":[],"status":"failed","title":"should not login with incorrect password"},{"ancestorTitles":["Auth Controller","POST /api/v1/auth/login"],"duration":2,"failureDetails":[{}],"failureMessages":["MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:220:13)\n    at NativeConnection.openUri (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\connection.js:766:34)\n    at Mongoose.connect (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\node_modules\\mongoose\\lib\\index.js:416:15)\n    at C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:21:20\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"],"fullName":"Auth Controller POST /api/v1/auth/login should not login with non-existent email","invocations":1,"location":null,"numPassingAsserts":0,"retryReasons":[],"status":"failed","title":"should not login with non-existent email"}],"endTime":1747963766326,"message":"\u001b[1m\u001b[31m  \u001b[1m● \u001b[22m\u001b[1mAuth Controller › POST /api/v1/auth/register › should register a new user\u001b[39m\u001b[22m\n\n    MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n\u001b[2m\u001b[22m\n\u001b[2m    \u001b[0m \u001b[90m 19 |\u001b[39m     \u001b[22m\n\u001b[2m     \u001b[90m 20 |\u001b[39m     \u001b[90m// Connect to the in-memory database\u001b[39m\u001b[22m\n\u001b[2m    \u001b[31m\u001b[1m>\u001b[22m\u001b[2m\u001b[39m\u001b[90m 21 |\u001b[39m     \u001b[36mawait\u001b[39m mongoose\u001b[33m.\u001b[39mconnect(mongoUri)\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m    |\u001b[39m                    \u001b[31m\u001b[1m^\u001b[22m\u001b[2m\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 22 |\u001b[39m   })\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 23 |\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 24 |\u001b[39m   afterEach(\u001b[36masync\u001b[39m () \u001b[33m=>\u001b[39m {\u001b[0m\u001b[22m\n\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.createClient (\u001b[22m\u001b[2mnode_modules/mongoose/lib/drivers/node-mongodb-native/connection.js\u001b[2m:220:13)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.openUri (\u001b[22m\u001b[2mnode_modules/mongoose/lib/connection.js\u001b[2m:766:34)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat Mongoose.connect (\u001b[22m\u001b[2mnode_modules/mongoose/lib/index.js\u001b[2m:416:15)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat \u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:21:20\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat fulfilled (\u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:5:58)\u001b[22m\u001b[2m\u001b[22m\n\n\u001b[1m\u001b[31m  \u001b[1m● \u001b[22m\u001b[1mAuth Controller › POST /api/v1/auth/register › should not register a user with duplicate email\u001b[39m\u001b[22m\n\n    MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n\u001b[2m\u001b[22m\n\u001b[2m    \u001b[0m \u001b[90m 19 |\u001b[39m     \u001b[22m\n\u001b[2m     \u001b[90m 20 |\u001b[39m     \u001b[90m// Connect to the in-memory database\u001b[39m\u001b[22m\n\u001b[2m    \u001b[31m\u001b[1m>\u001b[22m\u001b[2m\u001b[39m\u001b[90m 21 |\u001b[39m     \u001b[36mawait\u001b[39m mongoose\u001b[33m.\u001b[39mconnect(mongoUri)\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m    |\u001b[39m                    \u001b[31m\u001b[1m^\u001b[22m\u001b[2m\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 22 |\u001b[39m   })\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 23 |\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 24 |\u001b[39m   afterEach(\u001b[36masync\u001b[39m () \u001b[33m=>\u001b[39m {\u001b[0m\u001b[22m\n\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.createClient (\u001b[22m\u001b[2mnode_modules/mongoose/lib/drivers/node-mongodb-native/connection.js\u001b[2m:220:13)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.openUri (\u001b[22m\u001b[2mnode_modules/mongoose/lib/connection.js\u001b[2m:766:34)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat Mongoose.connect (\u001b[22m\u001b[2mnode_modules/mongoose/lib/index.js\u001b[2m:416:15)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat \u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:21:20\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat fulfilled (\u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:5:58)\u001b[22m\u001b[2m\u001b[22m\n\n\u001b[1m\u001b[31m  \u001b[1m● \u001b[22m\u001b[1mAuth Controller › POST /api/v1/auth/register › should not register a user with missing required fields\u001b[39m\u001b[22m\n\n    MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n\u001b[2m\u001b[22m\n\u001b[2m    \u001b[0m \u001b[90m 19 |\u001b[39m     \u001b[22m\n\u001b[2m     \u001b[90m 20 |\u001b[39m     \u001b[90m// Connect to the in-memory database\u001b[39m\u001b[22m\n\u001b[2m    \u001b[31m\u001b[1m>\u001b[22m\u001b[2m\u001b[39m\u001b[90m 21 |\u001b[39m     \u001b[36mawait\u001b[39m mongoose\u001b[33m.\u001b[39mconnect(mongoUri)\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m    |\u001b[39m                    \u001b[31m\u001b[1m^\u001b[22m\u001b[2m\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 22 |\u001b[39m   })\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 23 |\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 24 |\u001b[39m   afterEach(\u001b[36masync\u001b[39m () \u001b[33m=>\u001b[39m {\u001b[0m\u001b[22m\n\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.createClient (\u001b[22m\u001b[2mnode_modules/mongoose/lib/drivers/node-mongodb-native/connection.js\u001b[2m:220:13)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.openUri (\u001b[22m\u001b[2mnode_modules/mongoose/lib/connection.js\u001b[2m:766:34)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat Mongoose.connect (\u001b[22m\u001b[2mnode_modules/mongoose/lib/index.js\u001b[2m:416:15)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat \u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:21:20\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat fulfilled (\u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:5:58)\u001b[22m\u001b[2m\u001b[22m\n\n\u001b[1m\u001b[31m  \u001b[1m● \u001b[22m\u001b[1mAuth Controller › POST /api/v1/auth/login › should login an existing user with correct credentials\u001b[39m\u001b[22m\n\n    MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n\u001b[2m\u001b[22m\n\u001b[2m    \u001b[0m \u001b[90m 19 |\u001b[39m     \u001b[22m\n\u001b[2m     \u001b[90m 20 |\u001b[39m     \u001b[90m// Connect to the in-memory database\u001b[39m\u001b[22m\n\u001b[2m    \u001b[31m\u001b[1m>\u001b[22m\u001b[2m\u001b[39m\u001b[90m 21 |\u001b[39m     \u001b[36mawait\u001b[39m mongoose\u001b[33m.\u001b[39mconnect(mongoUri)\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m    |\u001b[39m                    \u001b[31m\u001b[1m^\u001b[22m\u001b[2m\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 22 |\u001b[39m   })\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 23 |\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 24 |\u001b[39m   afterEach(\u001b[36masync\u001b[39m () \u001b[33m=>\u001b[39m {\u001b[0m\u001b[22m\n\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.createClient (\u001b[22m\u001b[2mnode_modules/mongoose/lib/drivers/node-mongodb-native/connection.js\u001b[2m:220:13)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.openUri (\u001b[22m\u001b[2mnode_modules/mongoose/lib/connection.js\u001b[2m:766:34)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat Mongoose.connect (\u001b[22m\u001b[2mnode_modules/mongoose/lib/index.js\u001b[2m:416:15)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat \u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:21:20\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat fulfilled (\u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:5:58)\u001b[22m\u001b[2m\u001b[22m\n\n\u001b[1m\u001b[31m  \u001b[1m● \u001b[22m\u001b[1mAuth Controller › POST /api/v1/auth/login › should not login with incorrect password\u001b[39m\u001b[22m\n\n    MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n\u001b[2m\u001b[22m\n\u001b[2m    \u001b[0m \u001b[90m 19 |\u001b[39m     \u001b[22m\n\u001b[2m     \u001b[90m 20 |\u001b[39m     \u001b[90m// Connect to the in-memory database\u001b[39m\u001b[22m\n\u001b[2m    \u001b[31m\u001b[1m>\u001b[22m\u001b[2m\u001b[39m\u001b[90m 21 |\u001b[39m     \u001b[36mawait\u001b[39m mongoose\u001b[33m.\u001b[39mconnect(mongoUri)\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m    |\u001b[39m                    \u001b[31m\u001b[1m^\u001b[22m\u001b[2m\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 22 |\u001b[39m   })\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 23 |\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 24 |\u001b[39m   afterEach(\u001b[36masync\u001b[39m () \u001b[33m=>\u001b[39m {\u001b[0m\u001b[22m\n\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.createClient (\u001b[22m\u001b[2mnode_modules/mongoose/lib/drivers/node-mongodb-native/connection.js\u001b[2m:220:13)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.openUri (\u001b[22m\u001b[2mnode_modules/mongoose/lib/connection.js\u001b[2m:766:34)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat Mongoose.connect (\u001b[22m\u001b[2mnode_modules/mongoose/lib/index.js\u001b[2m:416:15)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat \u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:21:20\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat fulfilled (\u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:5:58)\u001b[22m\u001b[2m\u001b[22m\n\n\u001b[1m\u001b[31m  \u001b[1m● \u001b[22m\u001b[1mAuth Controller › POST /api/v1/auth/login › should not login with non-existent email\u001b[39m\u001b[22m\n\n    MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n\u001b[2m\u001b[22m\n\u001b[2m    \u001b[0m \u001b[90m 19 |\u001b[39m     \u001b[22m\n\u001b[2m     \u001b[90m 20 |\u001b[39m     \u001b[90m// Connect to the in-memory database\u001b[39m\u001b[22m\n\u001b[2m    \u001b[31m\u001b[1m>\u001b[22m\u001b[2m\u001b[39m\u001b[90m 21 |\u001b[39m     \u001b[36mawait\u001b[39m mongoose\u001b[33m.\u001b[39mconnect(mongoUri)\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m    |\u001b[39m                    \u001b[31m\u001b[1m^\u001b[22m\u001b[2m\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 22 |\u001b[39m   })\u001b[33m;\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 23 |\u001b[39m\u001b[22m\n\u001b[2m     \u001b[90m 24 |\u001b[39m   afterEach(\u001b[36masync\u001b[39m () \u001b[33m=>\u001b[39m {\u001b[0m\u001b[22m\n\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.createClient (\u001b[22m\u001b[2mnode_modules/mongoose/lib/drivers/node-mongodb-native/connection.js\u001b[2m:220:13)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat NativeConnection.openUri (\u001b[22m\u001b[2mnode_modules/mongoose/lib/connection.js\u001b[2m:766:34)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat Mongoose.connect (\u001b[22m\u001b[2mnode_modules/mongoose/lib/index.js\u001b[2m:416:15)\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat \u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:21:20\u001b[22m\u001b[2m\u001b[22m\n\u001b[2m      \u001b[2mat fulfilled (\u001b[22m\u001b[2m\u001b[0m\u001b[36msrc/test/authController.test.ts\u001b[39m\u001b[0m\u001b[2m:5:58)\u001b[22m\u001b[2m\u001b[22m\n","name":"C:\\Users\\hr80r\\CascadeProjects\\Sentivox\\backend\\src\\test\\authController.test.ts","startTime":1747963762330,"status":"failed","summary":""}],"wasInterrupted":false}
